tasks/main.yml

```
@@ -16,6 +20,11 @@
       set_fact:
         jetbrains_toolbox_versions: "{{ jetbrains_toolbox_available_releases.json.TBA | map(attribute='version') | list }}"
 
+    - name: Update version fact if latest is overridden
+      set_fact:
+        jetbrains_toolbox_version: "{{ jetbrains_toolbox_available_releases.json.TBA | sort(attribute='build', reverse=True) | map(attribute='version') | list | first }}"
+      when: latest
```

Something I've done in the past is reuse the version variable and check if it equals `latest`. If yes, then override the variable using `set_fact` like you've done here. Otherwise, it can stay set to the value it was already set to.

```suggestion
      when: jetbrains_toolbox_version == 'latest'
```

You could also remove lines 9-12 above going this route.

---

no worries! honestly, i'm just pumped that it got noticed. i've applied the suggested change, and updated the rest of the items, accordingly. thanks for taking the time!